referenceframe/frame_system.go

```diff
@@ -676,7 +671,7 @@ func createFramesFromPart(part *FrameSystemPart) (Frame, Frame, error) {
 
 	// Since the geometry of a frame system part is intended to be located at the origin of the model frame, we place it post-transform
 	// in the "_origin" static frame
-	return modelFrame, &tailGeometryStaticFrame{staticOriginFrame}, nil
+	return modelFrame, &tailGeometryStaticFrame{staticOriginFrame.(*staticFrame)}, nil
```

Comment:
Could we ever not be okay when  casting from a `staticOriginFrame` to a `staticFrame`?

Reply:
we'll always be fine because we only return staticFrames.  I could add back the check here to guard against someone in the future not realizing the function called `toStaticFrame` should always return a staticFrame

Reply:
It's fine as is, then.